core/src/main/java/com/sequenceiq/cloudbreak/service/publicendpoint/FreeIPAEndpointManagementService.java

```diff
@@ -43,10 +43,13 @@ public void registerLoadBalancerDomainWithFreeIPA(Stack stack) {
             try {
                 if (StringUtils.isNotEmpty(loadBalancer.getDns())) {
                     sendAddDnsCnameRecordRequest(stack, loadBalancer);
-                } else if (StringUtils.isNotEmpty(loadBalancer.getIp())) {
+                } else {
+                    LOGGER.error("Unable to find DNS for load balancer. Load balancer address will not be registered as CNAME with FreeIPA.");
```

Comment:
instead of creating a new `else` branch here you could add logs to every existing one and it should be obvious from the logs what happened

Reply:
This is just a reordering of some of my log messages. My preferences is to have explicit log messages, instead of trying to implicitly figure out what happened by looking for missing logs.

Reply:
The logic change `else if` to `if` here is to register the FQDN to IP mapping and reverse mapping for both AWS and Azure clusters. Without this change, Hue will not start.